packages/@aws-cdk/cloudformation-include/test/test-templates/functions-and-conditions.json

```diff
@@ -0,0 +1,104 @@
+{
+    "Mappings" : {
+      "RegionMap" : {
+        "region-1" : { 
+          "HVM64" : "name1", "HVMG2" : "name2" 
+        }
+      }
+    },
+    "Conditions": {
+        "AlwaysTrueCond": {
+          "Fn::Not": [{
+            "Fn::Equals": [
+              {
+                "Ref": "AWS::Region"
+              },
+              "completely-made-up-region"
+            ]
+          }]
+        },
+        "AlwaysFalseCond": {
+          "Fn::Equals": [
+            {
+              "Ref": "AWS::Region"
+            },
+            "completely-made-up-region"
+          ]
+        }
+      },
+    "Resources": {
+      "Vpc": {
+        "Type" : "AWS::EC2::VPC",
+        "Properties" : {
+          "CidrBlock" : {
+              "Fn::If": [
+                  "AlwaysTrueCond",
+                  { "Fn::Cidr": [ "192.168.1.1/24", "2"] },
+                  { "Fn::Cidr": [ "10.0.0.0/24", "6", "5"] }
+              ]
+          }
+        }
+      },
+      "Bucket": {
+        "Type": "AWS::S3::Bucket",
+        "Properties": {
+          "BucketName": {
+            "Fn::If": [
+              {"Fn::Or": ["AlwaysFalseCond", "AlwaysFalseCond"]},
+              "Unreachable",
+              {"Fn::FindInMap": [
+                "RegionMap",
+                "region-1",
+                "HVM64"
+              ]}
+            ]
+          }
+        }
+      },
+      "subnet1" : {
+        "Type" : "AWS::EC2::Subnet",
+        "Properties" : {
+          "VpcId" : { 
+            "Fn::If": [
+              {"Fn::And": ["AlwaysTrueCond", "AlwaysTrueCond"]},
```

Comment:
This is also not valid CloudFormation (`Fn::If` takes a Condition name as its first argument).

Reply:
You didn't address this comment.